openmc/data/reaction.py

```diff
@@ -163,11 +165,16 @@ def _get_products(ev, mt):
                     )
 
                 zat = ev.target["atomic_number"] * 1000 + ev.target["mass_number"]
-                projectile_mass = ev.projectile["mass"]
+                if ev.projectile['mass'] == 0.0:
+                    projectile_za = 0
+                elif np.isclose(ev.projectile['mass'], 1.0, atol=1.0e-12, rtol=0.):
```

Comment:
are these tolerances set uniformly across the data module? Maybe this should be elevated to some constant somewhere that is globally accessible?